patches/0002-Vendor-crypto-backends.patch

```diff
@@ -3,8 +3,11 @@ From: Quim Muntal <qmuntaldiaz@microsoft.com>
 Date: Mon, 23 May 2022 12:59:36 +0000
 Subject: [PATCH] Vendor crypto backends
 
-To reproduce, run 'go mod vendor' in 'go/src'.
+To reproduce changes in 'src/vendor', run 'go mod vendor' in 'src'.
```

Comment:
While here, maybe mention using a build of the current Go version to do the update.

```suggestion
To reproduce changes in 'src/vendor', run 'go mod vendor' in 'src'.
Use a 'go' that was recently built by the current branch to ensure stable results.
```

(https://github.com/golang/go/issues/69235)